tools/llm/utils.py

```diff
@@ -47,7 +47,45 @@ def export_llm(model, inputs, min_seq_len=1, max_seq_len=16):
     return ep
 
 
-def get_zeroed_static_cache_inputs(model: torch.fx.GraphModule):
+def export_llm_no_position_ids(model, inputs, min_seq_len=1, max_seq_len=16):
```

Comment:
why are we exporting without position_ids here ? Where is it used ? 
Consider merging this with `export_llm` by adding a use_position_ids argument to do this